app/components/admin/budgets/actions_component.rb

```diff
@@ -65,7 +65,17 @@ def ballots_action
                text: t("admin.budgets.actions.ballots"),
                path: create_budget_poll_path,
                method: :post,
-               confirm: t("admin.budgets.actions.confirm.ballots"))
+               confirm: t("admin.budgets.actions.confirm.ballots"),
+               disabled: !Setting["process.polls"])
+      end
+    end
+
+    def ballots_hint
+      if !Setting["process.polls"]
```

Comment:
Here the same comment about the feature?(name) method. 
And how about reversing the order of the condition and asking directly:
```
if feature?("process.polls")
  t("admin.budgets.actions.descriptions.ballots")
else
  link = admin_settings_path(anchor: "tab-participation-processes")
  t("admin.budgets.ballots.feature_disabled", link: link)
end
```

Reply:
Yeah, sounds good